packages/main/src/webComponents/Select/index.tsx

```diff
@@ -57,11 +56,12 @@ export interface SelectDomRef extends Omit<SelectAttributes, 'menu'>, Ui5DomRef
    */
   value?: string;
   /**
-   * Defines a reference (ID or DOM element) of the component's options menu, as an alternative to defining the selection's drop-down menu.
-   *
-   * **Note:** Usage of `SelectMenu` is recommended.
+   * Defines the value state of the component.
    */
-  menu?: string | HTMLElement;
```

Comment:
I think the menu getter from the dom ref can also be a HTML Element, can't it?

Reply:
Good catch!
I also forgot to move the `value` getter/setter to the DomRef again.